tests/e2e/e2e.mk

```
@@ -38,3 +38,25 @@ test-e2e-aws-simple-1-20: test-e2e-install
 		--test-package-marker=stable-1.20.txt \
 		--parallel 25 \
 		--skip-regex="\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|\[HPA\]|Dashboard|RuntimeClass|RuntimeHandler"
+
+.PHONY: test-e2e-do-simple-1-20
+test-e2e-do-simple-1-20: test-e2e-install
+	kubetest2 kops \
+		-v 6 \
+		--up --down \
+		--cloud-provider=digitalocean \
+		--cluster-name e2e-test-do.k8s.local \
+		--create-args networking=cilium \
+		--create-args api-loadbalancer-type=public \
+		--master-count=3 \
```

kubetest2-kops' --create-args is of type `string` rather than `[]string` so it can only be specified once. its value is space-separate flags though.

```suggestion
		--create-args "--networking=cilium --api-loadbalancer-type=public --master-count=3" \
```

---

Oh I see, got it. Thanks @rifelpet